src/main/resources/application.yml

```diff
@@ -20,6 +20,7 @@ worker:
   gasPriceMultiplier: ${IEXEC_GAS_PRICE_MULTIPLIER:1.3} # txs will be send with networkGasPrice*gasPriceMultiplier, 4.0 means super fast
   gasPriceCap: ${IEXEC_GAS_PRICE_CAP:22000000000} #in Wei, will be used for txs if networkGasPrice*gasPriceMultiplier > gasPriceCap
   overrideBlockchainNodeAddress: ${IEXEC_WORKER_OVERRIDE_BLOCKCHAIN_NODE_ADDRESS:} #will use it if set, else will use the one given by the core
+  developerLoggerEnabled: ${IEXEC_DEVELOPER_LOGGER_ENABLED:true}
```

Comment:
By default it should be false, we will enable it at the worker's level.

Reply:
The default value was for us when developing, but I can change it if you like yes

Reply:
Changed it*